test/helpers/position_assertions.h

```diff
@@ -22,12 +22,6 @@ int errorComparison(std::string_view aFilename, const Range &a, std::string_view
  */
 std::string prettyPrintRangeComment(std::string_view sourceLine, const Range &range, std::string_view comment);
 
-// Converts a relative file path into an absolute file:// URI.
-std::string filePathToUri(std::string_view prefixUrl, std::string_view filePath);
-
-// Converts a URI into a relative file path.
-std::string uriToFilePath(std::string_view prefixUrl, std::string_view uri);
```

Comment:
We seem to have duplicate header declarations for these helpers (here and in `helpers/lsp.h`)... so I removed these redundant declarations.